RELEASE_NOTES.md

```diff
@@ -31,75 +31,75 @@ Since darktable 4.4:
 - 49 issues closed
 
 
-_Please note that the darktable documentation is not currently complete for release 4.6
-and contributions are greatly appreciated. Please see the
-[project documentation](https://github.com/darktable-org/dtdocs#contributing)
-for more information on how to contribute._
-
 ## The Big Ones
 
-- n/a
+- Documentation is now complete for darktable 4.6 including translation
+  into Ukrainian and Dutch.
 
 ## Performance Improvements
 
-- Fix some OpenCL code paths that could produce garbled or unprocessed
+- Fixed some OpenCL code paths that could produce garbled or unprocessed
   output without back reporting the the pixel-pipe.
```

Comment:
BTW I don't really understand what "without back reporting the the pixel-pipe" is trying to say.

Reply:
I did understood it as "without reporting back to the pixel-pipe". Meaning the pixel-pipe is not aware of the OpenCL failure and could not take a proper decision like fallback on CPU path.

Reply:
Ok that makes more sense. I'll change it